addons/mail/models/mail_notification.py

```diff
@@ -79,6 +79,10 @@ def write(self, vals):
             vals['read_date'] = fields.Datetime.now()
         return super(Notification, self).write(vals)
 
+    @api.returns('self', lambda value: value.id)
```

Comment:
It's not necessary imo
PS: when you add a new translation in stable, you should export the .po file (in the technical view)